staging/cse/windows/kubeletfunc.ps1

```diff
@@ -209,6 +209,70 @@ function Get-KubePackage {
     Remove-Item $zipfile
 }
 

```

Comment:
It is better to add ut to cover these new lines

Reply:
added